websites/B/BoxSeries/presence.ts

```diff
@@ -0,0 +1,160 @@
+				if (!privacy && pathArray[4]) presenceData.state = titleMain;
+				presenceData.smallImageKey = "reading";
+				break;
+			case "anos":
+				presenceData.details = "Vendo anos";
+				presenceData.smallImageKey = "reading";
+				break;
+			case "release":
+				presenceData.details = "Vendo liberação";
+				if (!privacy && pathArray[4]) presenceData.state = titleMain;
+				presenceData.smallImageKey = "reading";
+				break;
+			case "series":
+				presenceData.details = "Vendo séries";
+				if (cover) {
+					presenceData.largeImageKey = document.querySelector<HTMLImageElement>(
+						"div > div > div > div.poster-season > img"
+					).src;
+				}
+				if (!privacy && pathArray[4]) {
+					presenceData.state = `${
+						document.querySelector("div > div > div > div.data > h1")
+							?.textContent ?? "pesquisa não encontrada"
+					}`;
+					presenceData.smallImageKey = "reading";
+				}
+				break;
+			case "episodio":
+				if (cover) {
+					presenceData.largeImageKey = document.querySelector<HTMLImageElement>(
+						"#dt_contenedor > div.season-bg-bs > div > a > img"
+					).src;
+				}
+				presenceData.details = `${titleEp[0]}`;
+				presenceData.state = `${titleEp[1]}`;
+				presenceData.smallImageKey = video.paused ? "pause" : "play";
+				presenceData.smallImageText = video.paused
+					? (await strings).pause
+					: (await strings).play;
+				if (!video.paused) {
+					[, presenceData.endTimestamp] = presence.getTimestamps(
+						Math.floor(video.current),
+						Math.floor(video.duration)
+					);
+				}
+				if (buttons) {
+					presenceData.buttons = [
+						{
+							label: "Assistindo episódio",
+							url: document.location.href.replace(/#\d+/, "")
+						}
+					];
+				}
+				break;
+			case "filmes":
+				if (cover) {
+					presenceData.largeImageKey = document.querySelector<HTMLImageElement>(
+						"#dt_contenedor > div.season-bg-bs > div > img"
+					).src;
+				}
+				presenceData.details = "Vendo filmes";
+				presenceData.state = `${
+					document.querySelector(
+						"#single > div.content.full_width_layout > div.dooplay_player > div.area-season.movie-bs > div.data > h1"
+					)?.textContent ?? "pesquisa não encontrada"
+				}`;
+				presenceData.smallImageKey = video.paused ? "pause" : "play";
+				presenceData.smallImageText = video.paused
+					? (await strings).pause
+					: (await strings).play;
+				if (!video.paused) {
+					[, presenceData.endTimestamp] = presence.getTimestamps(
+						Math.floor(video.current),
+						Math.floor(video.duration)
+					);
+				}
+				if (buttons) {
+					presenceData.buttons = [
+						{
+							label: "Assistindo filmes",
+							url: document.location.href.replace(/#\d+/, "")
+						}
+					];
+				}
+				break;
+			default:
+				presenceData.details = "página inicial";
+				presenceData.smallImageKey = "reading";
+				break;
+		}
+	}
+	if (!timestamps) {
+		delete presenceData.startTimestamp;
+		delete presenceData.endTimestamp;
+	}
+	if (privacy) {
```

Comment:
there are multiple `if (!privacy)` checks up above, please check those and adjust the code accordingly to remove redundancy. Also I don't think buttons should be disabled if privacy is disabled? If you do then the setting should have an if statement in the metadata file